.skiff/webapp.jsonnet

```diff
@@ -0,0 +1,235 @@
+                hosts: hosts
+            }
+        ],
+        rules: [
+            {
+                host: host,
+                http: {
+                    paths: [
+                        {
+                            backend: {
+                                serviceName: fullyQualifiedName,
+                                servicePort: config.httpPort
+                            }
+                        }
+                    ]
+                }
+            } for host in hosts
+        ]
+    }
+};
+
+// The `readinessProbe` begins on pod start and determines whether the pod can accept traffic.
+// When an application is not ready traffic will instead go to other ready pods.
+// A live pod can also become unready again if the probe fails `failureThreshold` times.
+local readinessProbe = {
+    failureThreshold: 9,
+    periodSeconds: 30,
+    initialDelaySeconds: 120,  // Use a longer delay if your app loads a large model.
+    httpGet: {
+        path: '/health?check=readiness',
+        port: config.httpPort,
+        scheme: 'HTTP',
+    },
+};
+
+// The `livenessProbe` begins on application start and determines whether to kill the pod.
+// If the `livenessProbe` fails `failureThreshold` times, the pod is killed and restarted by
+// Kubernetes.
+local livenessProbe = {
+    failureThreshold: 6,
+    periodSeconds: 10,
+    initialDelaySeconds: 120,  // Use a longer delay if your app loads a large model.
+    httpGet: {
+        path: '/health?check=liveness',
+        port: config.httpPort,
+        scheme: 'HTTP',
+    },
+};
+
+local deployment = {
+    apiVersion: 'extensions/v1beta1',
+    kind: 'Deployment',
+    metadata: {
+        labels: labels,
+        name: fullyQualifiedName,
+        namespace: namespaceName,
+    },
+    spec: {
+        revisionHistoryLimit: 3,
+        replicas: replicas,
+        template: {
+            metadata: {
+                name: fullyQualifiedName,
+                namespace: namespaceName,
+                labels: labels
+            },
+            spec: {
+                containers: [
+                    {
+                        name: config.appName,
+                        image: image,
+                        args: [ 'server/start.py', '--prod' ],
+                        readinessProbe: readinessProbe,
+                        livenessProbe: livenessProbe,
+                        resources: {
+                            requests: {
+                                // Our machines currently have 2 vCPUs, so this
+                                // will allow 4 apps to run per machine
+                                cpu: '1',
+                                // Each machine has 13 GB of RAM. We target 4
+                                // apps per machine, so we reserve 3 GB of RAM
+                                // for each (whether they use it our not).
+                                memory: '6Gi'
+                            }
```

Comment:
Should we try using less resources?

Reply:
Joel just upped this today out of need.  So probably not.